http-api/src/main/java/io/avaje/http/api/PathVariable.java

```diff
@@ -0,0 +1,21 @@
+package io.avaje.http.api;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/** Marks a method parameter to be a path variable. */
+@Retention(RUNTIME)
+@Target({PARAMETER, FIELD})
+public @interface PathVariable {
+
+  /**
+   * The name of the path variable.
+   *
+   * <p>If left blank the method parameter name is used.
+   */
+  String value();
```

Comment:
Not sure this should exist at all?

Reply:
Ok, I think we remove this for now as it isn't used in any tests and doesn't look strictly required.  I'll push that now.